server/data/prisonerClient.ts

```diff
@@ -11,9 +12,18 @@ export default class PrisonerClient {
     this.restClient = new RestClient('prisonerClient', config.apis.prisonerOffenderSearch as ApiConfig, token)
   }
 
-  async find(prisonNumber: string): Promise<Prisoner> {
-    return (await this.restClient.get({
-      path: prisonerOffenderSearchPaths.prisoner.show({ prisonNumber }),
-    })) as Prisoner
+  async find(prisonNumber: string, caseloadIds: Array<Caseload['caseLoadId']>): Promise<Prisoner | null> {
```

Comment:
Not from this PR, but `prisonNumber: string` -> `prisonNumber: Prisoner['prisonNumber']`

Is it worth making the other argument `caseLoadIds` here for consistency with the type?

Reply:
I've left `caseloads` as is (couldn't bear to have more `caseLoad`s) but updated the type of `prisonNumber`